phpseclib/Net/SSH2.php

```diff
@@ -3263,18 +3263,18 @@ private function get_binary_packet(bool $skip_channel_filter = false)
                     $cmf = ord($payload[0]);
                     $cm = $cmf & 0x0F;
                     if ($cm != 8) { // deflate
-                        user_error("Only CM = 8 ('deflate') is supported ($cm)");
+                        trigger_error("Only CM = 8 ('deflate') is supported ($cm)");
```

Comment:
These should be turned into Exceptions. This should be updated in the 3.0 branch too. I'll backport your changes to master!